src/components/overview/ProductInformation/ProductDesc.jsx

```diff
@@ -5,7 +5,9 @@ const ProductDesc = ({state, updateState}) => {
   let productFeature;
   if (state.selectedProductFeature) {
     productFeature = state.selectedProductFeature.map((element, index) => {
-      return <div key={index} style={{marginBottom: '2px'}}>{`- ${element.feature}: ${element.value}`}</div>;
+      return <div key={index} style={{marginBottom: '2px'}}>
+        <img id='checkMark' src='https://www.publicdomainpictures.net/pictures/130000/velka/check-mark-icon.jpg'></img>
```

Comment:
They're small images, so not sure it matters- but could you use an HTML check mark instead? An image might be better, incase the text checkmark doesn't render across all devises, I'm not sure.

Reply:
I did not know there's HTML check mark. I will definitely check it out!